omnibus/config/software/datadog-agent-integrations-py2.rb

```diff
@@ -62,6 +62,12 @@
   blacklist_packages.push(/^aerospike==/)  # Temporarily blacklist Aerospike until builder supports new dependency
 end
 
+if osx?
+  blacklist_folders.push('ibm_mq')
```

Comment:
💍

Reply:
would be worth adding a short comment explaining why we're blacklisting `ibm_mq`, `ibm_was` and the `lxml` dep ;)